### ISSUE TITLE
Can we cross compile it from Linux to ARM64?

### ISSUE THREAD

**Issue by Dravion** [PureDarwin/PureDarwin#131]
I wonder if it would be possible to Cross compile PureDarwin for ARM64 so we can run it in 
QEMU's aarch Emulator.

**Comment by wjk** [comment]
I don’t think so. xnu only supports Intel as far as I know; the ARM builds require Apple-internal-only tooling. In fact, the most recent version of xnu requires Apple-internal-only tooling for _all_ builds, making it impossible for us to use it. I have never tested ARM builds before.
